src/DynamoDb.DistributedLock/DynamoDb.DistributedLock.csproj

```diff
@@ -32,6 +32,7 @@
         <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="9.0.7" />
         <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="9.0.7" />
         <PackageReference Include="Microsoft.Extensions.Options" Version="9.0.7" />
+        <PackageReference Include="Microsoft.Extensions.Telemetry.Abstractions" Version="9.7.0" />
```

Comment:
🔄 Would prefer not to add any additional packages. Let's remove this and stick with what is available in `System.Diagnostics.DiagnosticSource`, already included in .Net 8+